app/data-sources/rural-payments-agency-org-applications-api.js

```diff
@@ -0,0 +1,17 @@
+import { RESTDataSource } from '@apollo/datasource-rest'
+
+export class RuralPaymentsAgencyOrganisationApplicationsAPI extends RESTDataSource {
```

Comment:
This should jsut be an additional method here: https://github.com/DEFRA/ffc-customer-registry-api/blob/main/app/data-sources/rural-payments-portal/RuralPaymentsPortalApi.js

Reply:
please see commit: [122cb13](https://github.com/DEFRA/ffc-customer-registry-api/pull/41/commits/122cb139c93da9b603f09456d9078840ce2c1878)